api/turing/models/router.go

```diff
@@ -60,17 +58,3 @@ func (r *Router) ClearCurrRouterVersion() {
 	r.ClearCurrRouterVersionID()
 	r.CurrRouterVersion = nil
 }
-
-// RouterResponse is an alias for the Router, to enable custom marshaling
-type RouterResponse Router
-
-// MarshalJSON is a custom marshaling function for the Router, to manipulate the
-// endpoint.
-func (r *Router) MarshalJSON() ([]byte, error) {
```

Comment:
removing it as it append `/v1/predict` for all cases and appending it where `http` is constructed instead in `api/turing/service/router_deployment_service.go`